Dockerfile

```diff
@@ -61,16 +61,16 @@ FROM alpine:3.17.1 as tools
 ARG TARGETARCH=amd64
 
 # renovate: datasource=github-releases depName=helm/helm
-ENV HELM_VERSION=v3.10.3
+ENV HELM_VERSION=v3.16.3
```

Comment:
let's just drop the changes to the dockerfile since it'll just merge conflict w/ my elixir/erlang upgrade pr.  I'll move them there.